Section_5.rb

```diff
@@ -0,0 +1,271 @@
+#======================================================================Intro to met
+#to define a method you must start with def
+def introduce_myself
+  puts "My name is Anndony Quemag"
+  puts "My age is 23"
+  puts "Im work at kommit"
+end
+#The method must be there before calling it, otherwise the code will have an error, as the method has no inputs, it is not necessary to put parentheses
+introduce_myself()
+introduce_myself
+puts
+#====================================================================Local variables
+#are defined within the definition of a method and this will only be executed while the method is executing
+last_name = "Quemag1"
+def introduce_myself
+  name = "Anndony"
+  last_name = "Quemag"
+  puts name
+  puts last_name
+end
+
+introduce_myself
+#puts name #generates error since the name variable does not exist global form
+puts last_name #they can have the same name as the local variables, but it tends to generate confusion
+#===================================================================Method parameters and arguments
+#are defined when creating the method in parentheses after the name
+def introduce_parameter(name, age)
+  puts "My name is #{name}"
+  puts "My age is "+age.to_s #{name}"
+end
+introduce_parameter("Anndony", 23)
```

Comment:
Try to create a new line when writing a new example to improve the reading when doing code review